rpc_jobs/params.yml

```diff
@@ -66,10 +66,10 @@
     parameters:
       - string:
           name: OSA_OPS_REPO
-          default: https://github.com/git-harry/openstack-ansible-ops
+          default: https://github.com/openstack/openstack-ansible-ops
       - string:
           name: OSA_OPS_BRANCH
-          default: 93ac83d3436fd078500d6fd85ec5078e357f4db4
+          default: master
```

Comment:
I think it'd be best if we use a SHA here to avoid upstream changes causing problems for us.

Reply:
Fixed